recipe-server/client/control_new/components/recipes/CloneRecipePage.js

```diff
@@ -0,0 +1,101 @@
+import { message } from 'antd';
+import autobind from 'autobind-decorator';
+import { Map } from 'immutable';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link, push as pushAction } from 'redux-little-router';
+
+import { SimpleLoadingOverlay } from 'control_new/components/common/LoadingOverlay';
+import RecipeForm from 'control_new/components/recipes/RecipeForm';
+import QueryRecipe from 'control_new/components/data/QueryRecipe';
+
+import { createRecipe as createAction } from 'control_new/state/recipes/actions';
+import { getRecipe } from 'control_new/state/recipes/selectors';
```

Comment:
You'll need to use the new `getRecipeForRevision` selector similar to:
https://github.com/mozilla/normandy/pull/858/files#diff-816a70a0a0eec30c1392dd3fabceb9eeR14